scripts/dts/gen_defines.py

```diff
@@ -40,7 +40,7 @@ def main():
 
     out_comment("Generated by gen_defines.py", blank_before=False)
     out_comment("DTS input file: " + args.dts, blank_before=False)
-    out_comment("Directory with bindings: " + args.bindings_dir,
+    out_comment("Directories with bindings: '{}'".format(args.bindings_dir),
```

Comment:
Could do `", ".join(args.binding_dirs)` maybe, to make it consistent with how lists are shown in some other places.

Reply:
Thinking

```python
out_comment("Directories with bindings: " + ", ".join(args.binding_dirs))
```